api/capi/src/nnstreamer-capi-util.c

```diff
@@ -1135,6 +1135,28 @@ ml_nnfw_to_accl_hw (const ml_nnfw_hw_e hw)
   }
 }
 
+/**
+ * @brief Internal function to convert accelerator as tensor_filter property format.
+ * @note returned value must be freed by the caller
+ * @note More details on format can be found in gst_tensor_filter_install_properties() in tensor_filter_common.c.
+ */
+char *
+ml_nnfw_to_str_prop (const ml_nnfw_hw_e hw)
+{
+  const gchar *hw_name;
+  const gchar *use_accl = "true:";
+  gchar *str_prop = NULL;
+
+  hw_name = get_accl_hw_str (ml_nnfw_to_accl_hw (hw));
+  str_prop = g_malloc (strlen(use_accl) + strlen(hw_name) + 1);
+  if (str_prop) {
+    strcpy(str_prop, use_accl);
```

Comment:
`strcpy` is allowed?

Reply:
replaced `strcpy` with `g_strdup_printf`.